classes/task/ReviewReminder.inc.php

```diff
@@ -56,70 +62,41 @@ public function sendReminder($reviewAssignment, $submission, $context, $reminder
-        $email->setContext($context);
-        $email->setReplyTo(null);
-        $email->addRecipient($reviewer->getEmail(), $reviewer->getFullName());
-        $email->setSubject($email->getSubject($context->getPrimaryLocale()));
-        $email->setBody($email->getBody($context->getPrimaryLocale()));
-        $email->setFrom($context->getData('contactEmail'), $context->getData('contactName'));
-
-        $reviewUrlArgs = ['submissionId' => $reviewAssignment->getSubmissionId()];
-        if ($reviewerAccessKeysEnabled) {
+        if ($reviewerAccessKeysEnabled) { // Give one-click access if enabled
             $accessKeyManager = new AccessKeyManager();
 
             // Key lifetime is the typical review period plus four weeks
             $keyLifetime = ($context->getData('numWeeksPerReview') + 4) * 7;
             $accessKey = $accessKeyManager->createKey($context->getId(), $reviewer->getId(), $reviewId, $keyLifetime);
-            $reviewUrlArgs = array_merge($reviewUrlArgs, ['reviewId' => $reviewId, 'key' => $accessKey]);
-        }
 
-        $application = Application::get();
-        $request = $application->getRequest();
-        $dispatcher = $application->getDispatcher();
-        $submissionReviewUrl = $dispatcher->url($request, PKPApplication::ROUTE_PAGE, $context->getPath(), 'reviewer', 'submission', null, $reviewUrlArgs);
-
-        // Format the review due date
-        $reviewDueDate = strtotime($reviewAssignment->getDateDue());
-        $dateFormatShort = $context->getLocalizedDateFormatShort();
-        if ($reviewDueDate === -1 || $reviewDueDate === false) {
-            // Default to something human-readable if no date specified
-            $reviewDueDate = '_____';
-        } else {
-            $reviewDueDate = strftime($dateFormatShort, $reviewDueDate);
-        }
-        // Format the review response due date
-        $responseDueDate = strtotime($reviewAssignment->getDateResponseDue());
-        if ($responseDueDate === -1 || $responseDueDate === false) {
-            // Default to something human-readable if no date specified
-            $responseDueDate = '_____';
-        } else {
-            $responseDueDate = strftime($dateFormatShort, $responseDueDate);
+            $reviewUrlArgs = ['submissionId' => $reviewAssignment->getSubmissionId(), 'reviewId' => $reviewId, 'key' => $accessKey];
+            $submissionReviewUrl = $dispatcher->url($request, PKPApplication::ROUTE_PAGE, $context->getPath(), 'reviewer', 'submission', null, $reviewUrlArgs);
+            $mailable->addData(['submissionReviewUrl' => $submissionReviewUrl]);
         }
-        $paramArray = [
-            'recipientName' => $reviewer->getFullName(),
-            'recipientUsername' => $reviewer->getUsername(),
-            'reviewDueDate' => $reviewDueDate,
-            'responseDueDate' => $responseDueDate,
-            'signature' => $context->getData('contactName') . "\n" . $context->getLocalizedName(),
+
+        $oldVariables = [
```

Comment:
Is there a better name we can use for these? Or if they are old, add a comment describing why they are old. Are they deprecated? Or what makes them that way?

Reply:
Yes, deprecated